go.mod

```diff
@@ -62,6 +62,7 @@ require (
 
 require (
 	cloud.google.com/go/compute/metadata v0.5.0 // indirect
+	dario.cat/mergo v1.0.1 // indirect
```

Comment:
how did this new dependency come about? I can't see us using this

Reply:
that shall be removed as I refactored it. dependency is not needed anymore.